src/fcm_api_v1.erl

```diff
@@ -62,9 +62,9 @@ do_push(RegId, Message0, AuthKey, PushUrl) ->
         Error -> Error
     end.
 
-reload_access_token(#{service_file := ServiceFile} = State) ->
+reload_access_token(#{service_file := ServiceFile}) ->
     {ok, Bin} = file:read_file(ServiceFile),
-    reload_access_token(State#{service_file_bin => Bin});
+    reload_access_token(#{service_file_bin => Bin});
```

Comment:
in case additional things have been added/are added in the future to the state:
```suggestion
    reload_access_token(maps:remove(service_file, State#{service_file_bin => Bin}));
```